source/tool/tests/hardware/test_x1_carbon_ubuntu.py

```diff
@@ -0,0 +1,22 @@
+from tests.hardware import test_ubuntu
+
+class X1CarbonUbuntuTest(test_ubuntu.GenericUbuntuTest):
+
+    PRODUCT_NAME = '20FCS00Y01'
+    BIOS_VERSION = 'N1FET34W (1.08 )'
+    PASS = ["chipsec.modules.common.spi_desc",
+            "chipsec.modules.common.bios_wp",
+            "chipsec.modules.common.spi_lock",
+            "chipsec.modules.common.smrr",
+            "chipsec.modules.common.bios_ts",
+            "chipsec.modules.common.smm",
+            "chipsec.modules.common.secureboot.variables",
+            "chipsec.modules.common.uefi.access_uefispec",
+            "chipsec.modules.common.bios_smi",
+            "chipsec.modules.smm_dma",
+            "chipsec.modules.remap"]
+
+    SKIPPED = ["chipsec.modules.module_template"]
+
```

Comment:
Is there a reason you don't have super.setUp() and super.tearDown() like you do in [test_z420_ubuntu.py](https://github.com/chipsec/chipsec/blob/master/source/tool/tests/hardware/test_z420_ubuntu.py)?

I see that in [test_elitebook_1040_ubuntu.py](https://github.com/chipsec/chipsec/blob/master/source/tool/tests/hardware/test_elitebook_1040_ubuntu.py) you only have super.setUp but not tearDown. If this is on purpose can you please add comment in relevant files to explain why :)